examples/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ add_subdirectory(fly_multiple_drones)
 add_subdirectory(fly_qgc_mission)
 add_subdirectory(follow_me)
 add_subdirectory(geofence_inclusion)
+add_subdirectory(gimbal)
```

Comment:
`gimbal_control` maybe?

Reply:
Yer, I was wondering that too. Often we just name the example the same as the plugin, so from that perspective it's fine.